.github/workflows/ci.yaml

```
@@ -0,0 +1,30 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'master'
```

```suggestion
on:
  workflow_dispatch:
  push:
    branches:
      - 'master'
```

This will allow us to build+push a new image manually (if we need to). We should also take into consideration which branch this is being built on:
| Branch | Tag |
|--------|-----|
|`master`|`latest`|
|`release/v0.1.0`|`v0.1.0`|